### DISCUSSION TITLE
Adding reverse edges as new edge types

### DISCUSSION THREAD

**Discussion by tehranixyz** [pyg-team/pytorch_geometric#6029] [Q&A]
Hi,

I have a custom dataset of homogeneous graphs. 
Each graph has five types of edges with edge attributes. (x, edge_index, edge_type, edge_attr)
Is there any way I can add reverse edges as a new type of edge?
Let's say I have edge types of 'friend', and 'relative'; I want the reverse edges to have types 'friend_rev' and 'relative_rev'.

Thanks.

**Comment by rusty1s** [comment]
```python
rev_edge_index = edge_index[:, edge_type = xxx].flip([0])
edge_index = torch.cat([edge_index, rev_edge_index], dim=1)
```
